Tribute/Tribute.swift

```diff
@@ -58,6 +58,8 @@ public struct Attributes {
     public var hyphenationFactor: Float?
     public var allowsTighteningForTruncation: Bool?
     
+    private var paragraph:NSParagraphStyle?
+    private static let defaultParagraph = NSParagraphStyle.defaultParagraphStyle()
```

Comment:
this call is fairly fast: first time it will `alloc] init]` instance; all subsequential calls will return ivar